src/components/AIAssistant/AIAssistant.tsx

```
@@ -261,18 +261,21 @@ export const AIAssistant = () => {
                 className={cn(
-                  'mb-4 p-3 rounded-lg',
+                  'mb-4 p-3 rounded-lg break-words max-w-[85%]',
                   message.role === 'user'
                     ? 'bg-primary text-primary-foreground ml-auto'
                     : 'bg-muted',
                 )}
               >
-                {message.content}
+                <p
+                  className="text-sm"
+                  dangerouslySetInnerHTML={{ __html: message.content.replace(/\n/g, '<br />') }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)